src/components/ebay-toggle-button-group/toggle-button-group.stories.js

```diff
@@ -0,0 +1,125 @@
+import { tagToString } from "../../../.storybook/storybook-code-source";
+import { addRenderBodies } from "../../../.storybook/utils";
+import readme from "./README.md";
+import component from "./index.marko";
+import withIconsTemplate from "./examples/icons.marko";
+import withIconsCode from "./examples/icons.marko?raw";
+import withDefaultTemplate from "./examples/withDefault.marko";
+import withDefaultCode from "./examples/withDefault.marko?raw";
+import controlledTemplate from "./examples/controlled.marko";
+import controlledCode from "./examples/controlled.marko?raw";
+
+const Template = (args) => ({
+    input: addRenderBodies(args),
+});
+
+export default {
+    title: "buttons/ebay-toggle-button-group",
+    component,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        variant: {
+            type: "string",
+            control: { type: "select" },
+            options: ["checkbox", "radio", "radio-toggle"],
+            description:
+                'Selection type for the buttons in the group. May be `"checkbox"` (default), `"radio"`, or `"radio-toggle"` (same as radio but with the option to deselect)',
+        },
+        columns: {
+            type: "number",
+            control: { type: "number" },
+            description: "Preferred minimum number of columns",
+        },
+        layoutType: {
+            type: "string",
+            control: { type: "select" },
+            options: ["minimal", "list", "gallery"],
+            description:
+                'Enforced layout type of all buttons. May be `"minimal"` (default), `"list"`, or `"gallery"`. Gallery layout may only be used when there is also an icon or an image, and minimal layout may **not** be used when there is an icon or an image',
+        },
+        buttons: {
+            name: "@button",
+            description:
+                "Represents an `<ebay-toggle-button/>` to be used as part of the group",
+            table: {
+                category: "@attribute tags",
+            },
+        },
+        onChange: {
+            action: "on-change",
+            description: "Triggered when the pressed state changes",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, pressed }",
+                },
+            },
+        },
+    },
+};
+
+export const Default = Template.bind({});
+Default.args = {
+    columns: 3,
+    buttons: [
+        { title: "Button 1" },
+        { title: "Button 2" },
+        { title: "Button 3" },
+        { title: "Button 4" },
+        { title: "Button 5" },
+        { title: "Button 6" },
```

Comment:
Lets also change these.

Reply:
Ok, updated. I also snuck in a little title update for one of the storybook pages.